File: Python/08 - Clases/Miguel753Dav9.py
class Programmer:
    
    surname: str = None
    def __init__(self, name: str, age: int, lenguage: list):
        self.name = name
        self.age = age
        self.lenguage = lenguage

    def print(self):
            print(
                 f"Nombre: {self.name} | Apellidos: {self.surname} | Edad: {self.age} | Lenguajes: {self.lenguage}")

my_programmer = Programmer("Miguel", 11, ["Python", "Java"])
my_programmer.print()
my_programmer.surname = "Pallares"
my_programmer.print()
my_programmer.age = 12
my_programmer.print()

File: Java/src/Topic08_Classes/Miguel753Dav9.java
package Topic08_Classes;

import java.util.ArrayList;

public class Miguel753Dav9 {
    String name;
    String surname;
    int age;
    ArrayList<String> Lenguajes = new ArrayList<>();

    public Miguel753Dav9(String name, String surname, int age, ArrayList Lenguajes) {
        this.name = name;
        this.surname = surname;
        this.age = age;
        this.Lenguajes = Lenguajes;


    }
    public String printInfo() {
        return name + " " + surname + ", Edad: " + age + ",  Lenguajes de programacion: " + Lenguajes;
    }
    public static void main(String[] args){
        ArrayList<String> temp = new ArrayList<>();
        temp.add("Python");
        temp.add("Java");
        Miguel753Dav9 objProgrammer = new Miguel753Dav9("Miguel", "Pallares", 12, temp);
        System.out.println(objProgrammer.printInfo());
    }
}

